public/testdata/flight_number_by_state.csv

```diff
@@ -0,0 +1,18 @@
+State, Lat,Lng,Count
```

Comment:
rather than having two files, can we just add two fields to the same file? and maybe call it something like: flight_info_by_state.csv
```
State,Lat,Lng,DestLocation,Count,Price
```

If we get bold, we can add an [h3](https://h3geo.org/) column -- the much better improvement to geohash